addOns/replacer/src/main/java/org/zaproxy/zap/extension/replacer/ExtensionReplacer.java

```diff
@@ -136,6 +138,61 @@ private static boolean contains(String original, String match, Pattern p) {
     }
 
     private static String replace(String original, String match, Pattern p, String replacement) {
+        return replace(original, match, p, replacement, false);
+    }
+
+    private static String replace(
+            String original, String match, Pattern p, String replacement, boolean extraProcessing) {
+        LOGGER.debug("Static Replacement function.");
+
+        if (extraProcessing) {
+            int specialStart = replacement.indexOf("{{");
+            if (specialStart >= 0) {
+                LOGGER.debug("Special replacement detected.");
+                int specialEnd = replacement.indexOf("}}") + 2;
```

Comment:
For the case of MIN/MAX with non-integer values, what would you view as the correct behaviour?

I can think of two simple possibilities:
- Invalid values are ignored but the base replacement happens.
- Invalid values cause the entire replacement to be ignored.

Reply:
I'd go with the latter (more obvious something is wrong, I think), having said that, ideally it would be validated/rejected when the rule is being added rather than fail when used.